.github/workflows/main.yaml

```diff
@@ -26,7 +31,7 @@ jobs:
           pip install tox
 
       - name: Check code quality with flake8
-        run: tox -e flake8
+        run: tox -e lint
 
   test:
     runs-on: ubuntu-latest
```

Comment:
Below, also inject poetry dynamic versioning into pipx-installed poetry:
```yaml
  - name: Install Poetry
      run: |
        pipx install poetry
        pipx inject poetry poetry-dynamic-versioning
```

Reply:
Versioning isn't really needed to run the tests, is it?

Reply:
From the run logs a version 0.0.0 is installed. While it does not break tests, you will also not notice if dynamic versioning breaks or is configured wrong, if version 0.0.0 is "good enough".